src/wallet/rpcdisclosure.cpp

```diff
@@ -160,10 +160,10 @@ UniValue z_validatepaymentdisclosure(const UniValue& params, bool fHelp)
             "\nEXPERIMENTAL FEATURE\n"
             + strPaymentDisclosureDisabledMsg +
             "\nArguments:\n"
-            "1. \"paymentdisclosure\"     (string, required) Hex data string\n"
+            "1. \"paymentdisclosure\"     (string, required) Hex data string, with \"zpd:\" prefix.\n"
             "\nExamples:\n"
-            + HelpExampleCli("z_validatepaymentdisclosure", "\"hexblob\"")
-            + HelpExampleRpc("z_validatepaymentdisclosure", "\"hexblob\"")
+            + HelpExampleCli("z_validatepaymentdisclosure", "\"paymentdisclosure\"")
+            + HelpExampleRpc("z_validatepaymentdisclosure", "\"paymentdisclosure\"")
```

Comment:
These should probably have prefixes, although if this is meant to be analogous to the help text, then the words wouldn't have the prefix, so this would instead have to be a prefix followed by plausible-looking payment disclosure content.

Reply:
Done